File: src/ServerPulse.Backend/AnalyzerApi/Domain/Dtos/ServerLoadResponse.cs
namespace AnalyzerApi.Domain.Dtos
{
    public class ServerLoadResponse
    {
        public string Id { get; set; }
        public DateTime CreationDateUTC { get; set; }
        public string Endpoint { get; set; }
        public string Method { get; set; }
        public int StatusCode { get; set; }
        public TimeSpan Duration { get; set; }
        public DateTime TimestampUTC { get; set; }
    }
}

File: src/ServerPulse.Frontend/src/app/modules/shared/domain/dtos/statistics/serverLoadResponse.ts
import { TimeSpan } from "../../..";

export interface ServerLoadResponse {
    id: string;
    creationDateUTC: Date;
    endpoint: string;
    method: string;
    statusCode: number;
    duration: TimeSpan;
    timestampUTC: Date;
}
export function convertToServerLoadResponse(data: any): ServerLoadResponse {
    return {
        id: data.Id,
        creationDateUTC: data.CreationDateUTC,
        endpoint: data.Endpoint,
        method: data.Method,
        statusCode: data.StatusCode,
        duration: data.Duration,
        timestampUTC: data.TimestampUTC,
    };
}